api/src/certification/enrolment/domain/usecases/has-been-candidate.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * @typedef {import('./index.js').CandidateRepository} CandidateRepository
+ */
+
+/**
+ * @param {Object} params
+ * @param {number} params.userId
+ * @param {CandidateRepository} params.candidateRepository
+ */
+export async function hasBeenCandidate({ userId, candidateRepository }) {
```

Comment:
Non-bloquant: 

Daprès le code, on veut surtout savoir ici si en tant que candidat il a déjà rejoint une session

```suggestion
export async function hasJoinedASession({ userId, candidateRepository }) {

Reply:
@P-Jeremy  On avait vu ce nom avec @Steph0. Peut-être vous pouvez en rediscuter avec l'équipe pour voir quel serait le meilleur nom ?

Reply:
Non c'est cool, si ça a déjà été vu en amont, je valide 😉